examples/contracts/ERC20/ERC20.rs

```diff
@@ -0,0 +1,210 @@
+// Generated with Sol2Ink v0.3.0
+// https://github.com/Supercolony-net/sol2ink
+
+#![cfg_attr(not(feature = "std"), no_std)]
+#![feature(min_specialization)]
+
+#[brush::contract]
+pub mod erc_20 {
+	use brush::traits::AccountId;
+	use ink::prelude::string::String;
+	use ink_storage::Mapping;
+
+	#[ink(event)]
+	pub struct Transfer {
+		#[ink(topic)]
+		from: AccountId,
+		#[ink(topic)]
+		to: AccountId,
+		value: u128,
+	}
+
+	#[ink(event)]
+	pub struct Approval {
+		#[ink(topic)]
+		owner: AccountId,
+		#[ink(topic)]
+		spender: AccountId,
+		value: u128,
+	}
+
+	pub enum Enum {
+		FIRST, 
+		SECOND, 
+	}
+
+	#[derive(Default, Encode, Decode)]
+	#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
+	pub struct Struct  {
+		pub field_1: u128,
+		pub field_2: u128,
+	}
+
+	#[ink(storage)]
+	#[derive(Default, SpreadAllocate)]
+	pub struct  ERC20 {
+		balances: Mapping<AccountId, u128>,
+		allowances: Mapping<(AccountId, AccountId), u128>,
+		total_supply: u128,
+		name: String,
+		symbol: String,
+	}
+	
+	impl  ERC20 {
```

Comment:
it can be 1 space between impl and ERC20. Maybe exist a way automatically call cargo fmt after generating